src/wms/src/main/java/org/geoserver/wms/map/MetatileMapOutputFormat.java

```diff
@@ -37,6 +37,9 @@
 import org.geotools.resources.i18n.Errors;
 import org.geotools.util.logging.Logging;
 
+import it.geosolutions.jaiext.BufferedImageAdapter;
+import it.geosolutions.rendered.viewer.RenderedImageBrowser;
```

Comment:
oversight: RenderedImageBrowser has been probably imported during some interactive debug :) I would remove it.